### PR TITLE
PAAS-1325: allow Workato & IT OVH firewall IP addresses

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-1325

Short description:

### CODE REVIEW

File: packages/jcustomer/migrations/migrate-to-v4.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+globals:
+  version: 4
+  workato_ip_adds: "52.5.142.59,34.226.132.221,52.54.43.157,34.204.129.29,34.228.172.35,54.83.143.113,34.192.94.13,34.195.128.7,34.226.84.130"
+  it_ovh_firewall_add: "188.165.59.149"
+
+####
+# Ansible has a weird behavior when receiving a variable in its parameters,
+# it replaces " with ' which leads to an invalid json. To avoid the bug pass the parameters package_settings as follow :
+# -e "package_settings=' {\"papi_hostname\":\"papi[.dev].trial.jahia.com\",\"papi_token\":\"TMP_TOKEN\", \"status\": \"{{status }}\"}'"
+####
```
Comment (Ronan-Gorain) [inline packages/jcustomer/migrations/migrate-to-v4.yaml:22]: Useless as there is no need for parameters here

File: packages/jcustomer/install.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ globals:
   jcustomer_env_version: 3
```
Comment (kevinvandel) [inline packages/jcustomer/install.yml:16]: Since you created the "migrate-to-v4" package, you also need to increment the `jcustomer_env_version` here.

**Review by kevinvandel — DISMISSED** [review]

File: packages/jcustomer/migrations/migrate-to-v4.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        container_env_vars = jelastic.env.control.GetContainerEnvVars(
+                                     "${env.envName}",
+                                     session,
+                                     ${nodes.cp.first.id}
+                                   );
+        allowed_ips = container_env_vars.object.UNOMI_THIRDPARTY_PROVIDER1_IPADDRESSES;
+        if (allowed_ips != "") {
+          allowed_ips = "," + allowed_ips
+        }
+        return {"result": 0, "UNOMI_THIRDPARTY_PROVIDER1_IPADDRESSES": "127.0.0.1,::1,${globals.workato_ip_adds},${globals.it_ovh_firewall_add}," + allowed_ips}
```
Comment (Ronan-Gorain) [inline packages/jcustomer/migrations/migrate-to-v4.yaml:70]: Unless I'm mistaken, the env var still needs to be updated. Also, maybe you just have to call the api in the script, no need to return it.

**Review by Ronan-Gorain — CHANGES_REQUESTED** [review]

File: packages/jcustomer/migrations/migrate-to-v4.yaml (hunk 3 above)
Comment (adoux) [reply packages/jcustomer/migrations/migrate-to-v4.yaml]: Very good catch, forgot the essential part :facepalm: 
Fixed

File: packages/jcustomer/migrations/migrate-to-v4.yaml (hunk 1 above)
Comment (adoux) [reply packages/jcustomer/migrations/migrate-to-v4.yaml]: Removed!

File: packages/jcustomer/install.yml (hunk 2 above)
Comment (adoux) [reply packages/jcustomer/install.yml]: Good catch, fixed!

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**